lib/evmone/tracing.cpp

```diff
@@ -70,39 +71,36 @@ class InstructionTracer : public Tracer
 {
     struct Context
     {
+        const int32_t depth;
         const uint8_t* const code;  ///< Reference to the code being executed.
         const int64_t start_gas;
 
-        Context(const uint8_t* c, int64_t g) noexcept : code{c}, start_gas{g} {}
+        Context(int32_t d, const uint8_t* c, int64_t g) noexcept : depth{d}, code{c}, start_gas{g}
+        {}
     };
 
     std::stack<Context> m_contexts;
     std::ostream& m_out;  ///< Output stream.
+    std::ostringstream m_buf;
```

Comment:
This improved performance ~2x but I'm not sure if this is the best option. Maybe there is an option to tune `clog` itself. I'm thinking of omitting this change for now.

@gumb0 do you have any hints about iostream performance?

Reply:
We change the default `clog` buffer to `std::ofstream` in `t8n` implementation to redirect the output to file (to satisfy retesteth requirements). No sure then how the suggested change with `clog` tuning will work here. We probably would have to tune `std::ofstream` buffer

Reply:
Yes, omit this change for now.